src/main/java/org/snomed/snowstorm/core/data/domain/Annotation.java

```diff
@@ -0,0 +1,137 @@
+package org.snomed.snowstorm.core.data.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonView;
+import io.micrometer.core.instrument.util.StringUtils;
+import org.snomed.snowstorm.core.pojo.TermLangPojo;
+import org.snomed.snowstorm.rest.View;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@JsonView(value = View.Component.class)
+@JsonIgnoreProperties({"memberId", "additionalFields", "effectiveTimeI", "deleted", "changed"})
```

Comment:
Thanks for hiding these properties from the api response @QuyenLy87 however no other domain object seems to use the `JsonIgnoreProperties` annotation. Can you hide these properties without using the annotation so it is consistent across the domain?

Reply:
Hi @dmcgihtsdo, The annotation is actually a refset but has a different form. So it will extend the class "ReferenceSetMember". And because it does not extend SnomedComponent,so those properties cannot be hidden by using "@JsonView(value = View.Component.class)". That's why I have to hide them by JsonIgnoreProperties.